executable/executable.go

```diff
@@ -233,6 +258,10 @@ func (e *Executable) Wait() (ExecutableResult, error) {
 func (e *Executable) Kill() error {
 	doneChannel := make(chan error, 1)
 
+	if e.cmd.Process == nil {
```

Comment:
I'd place `doneChannel` after this line - early returns should be placed as "early" as possible, so that there's no confusion around why they're placed later in the flow.